LDAR_Sim/src/virtual_world/infrastructure.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, virtual_world, methods, in_dir) -> None:
         self.repair_delay_dataframe: pd.DataFrame = read_in_repair_delay_sources_file(
             inputs_path=in_dir, virtual_world=virtual_world
         )
+        self._sites: list[Site] = []
```

Comment:
Question: Is this just for cleaner code practices where all class variables are declared in the constructor?

Reply:
For the infrastructure specifically, it provides cleaner code, and is also I would say the most important class variable for this class.

Reply:
Can we be consistent and change the other classes to declare all instance variables as well then?